src/store/reducers/packagesReducer.js

```diff
@@ -48,7 +56,15 @@ export const epics = createEpicScenario({
       ConfigService.deletePackage(fromAction.payload)
         .map(toActionCreator(redux.actions.deletePackage, fromAction))
         .catch(handleError(fromAction))
-  }
+  },
+  /** Gets configuration types*/
+  fetchConfigTypes: {
+    type: 'CONFIG_TYPE_FETCH',
```

Comment:
Should this have "PACKAGES_" in the type name since its part of the packages reducer?

Reply:
done